src/utils/dateUtils.js

```diff
@@ -223,6 +223,46 @@ const DateUtils = {
         });
     },
 
+    timeFromNow(
+        date,
+        future = 'in %s',
+        past = '%s ago',
+        s = 'a few seconds',
+        ss = '%d seconds',
+        m = 'a minute',
+        mm = '%d minutes',
+        h = 'an hour',
+        hh = '%d hours',
+        d = 'a day',
+        dd = '%d days',
+        M = 'a month',
+        MM = '%d months',
+        y = 'a year',
+        yy = '%d years',
+    ) {
+        // Customizing moment's relativeTime
+        moment.updateLocale('en', {
+            relativeTime: {
+                future,
+                past,
+                s,
+                ss,
+                m,
+                mm,
+                h,
+                hh,
+                d,
+                dd,
+                M,
+                MM,
+                y,
+                yy,
+            },
+        });
```

Comment:
I imagine this actually mutates some state inside the global `moment` function/object definition?  Are we okay with that?  We've probably been doing it all along, so it's probably okay.

Reply:
It does, but using Moment's `fromNow` function I am not sure how else to go about it. Do you have any ideas?

Reply:
Nah, not really.  It seems we essentially want to customize this part of moment's locale settings.  In the future, we might consider some research into how moment handles these locale settings and maybe find some way to set these locale settings once, and not _every time_ our `timeFromNow()` function is called.  But that doesn't have to block this PR.